.build/Dockerfile

```diff
@@ -311,31 +344,33 @@ LABEL authors="Christoph Schranz <christoph.schranz@salzburgresearch.at>, Mathem
 
 USER root
 
-RUN pip install --no-cache-dir ipyleaflet plotly==4.8.* "ipywidgets>=7.5"
+RUN pip install --no-cache-dir ipyleaflet "plotly>=4.14.3" "ipywidgets>=7.5"
 
 # Install important packages and Graphviz
 RUN set -ex \
  && buildDeps=' \
     graphviz==0.11 \
 ' \
  && apt-get update \
- && apt-get -y install htop apt-utils graphviz libgraphviz-dev openssh-client \
+ && apt-get -y install htop apt-utils iputils-ping graphviz libgraphviz-dev openssh-client \
  && pip install --no-cache-dir $buildDeps
 
 # Install various extensions
 RUN fix-permissions $CONDA_DIR
-RUN jupyter labextension install @jupyterlab/github
-RUN jupyter labextension install jupyterlab-drawio
-RUN jupyter labextension install jupyter-leaflet
-RUN jupyter labextension install jupyterlab-plotly@4.8.1
-RUN jupyter labextension install @jupyter-widgets/jupyterlab-manager
-RUN pip install --no-cache-dir jupyter-tabnine==1.0.2  && \
-  jupyter nbextension install --py jupyter_tabnine && \
-  jupyter nbextension enable --py jupyter_tabnine && \
-  jupyter serverextension enable --py jupyter_tabnine
+
+# RUN jupyter labextension install @jupyterlab/github
+# RUN pip install jupyterlab-git
+RUN pip install jupyterlab-drawio
```

Comment:
Are all of these installed by default? Drawio and plotly?

Reply:
yes